Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_finalny_model.ipynb

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
<p>To co mówiłem, fajnie jakbyście jakoś&nbsp;podparły wybór tych cech. Może auto feature selection, ale to powinno być zrobione przed wcześniej. Może to też być VIF albo drzewo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>